spec/lib/link_check_spec.rb

```diff
@@ -0,0 +1,228 @@
+require "uri_checker"
```

Comment:
If you put `lib/` in the eager load path you'll not need to require these explicitly (which is also a problem you'll have in the app.